runtime/compiler/control/RecompilationInfo.hpp

```diff
@@ -87,6 +87,7 @@ class TR_PersistentMethodInfo
    public:
    TR_PERSISTENT_ALLOC(TR_Memory::PersistentMethodInfo);
 
+   TR_PersistentMethodInfo() {}
```

Comment:
Where is this constructor used? Generally speaking, constructors that don't construct much are dangerous.

Reply:
It's used to construct `CompilationInfoPerThreadRemote::_recompilationMethodInfo`. 

https://github.com/eclipse/openj9/blob/42185c596514d8954debd5bdc1e706c119a68adb/runtime/compiler/control/JITServerCompilationThread.cpp#L393